integrationTests/cypress/integration/redmine_administration.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Administrative Procedures
+
+  @requires_testuser
+  Scenario: cas user + no redmine user => login => create normal redmine account
+    Given the user is not member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + no redmine user => login => create admin redmine account
+    Given the user is member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with admin privileges
+
+  @requires_testuser
+  Scenario: cas user + internal default redmine user => login => preserves default account
+    Given the user is not member of the admin user group
+    And the user has an internal default redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user + internal cas-admin redmine user => login => demote to default redmine account
+    Given the user is not member of the admin user group
+    And the user has an internal admin redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + internal default redmine user => login => promote to cas-admin redmine account
+    Given the user is member of the admin user group
+    And the user has an internal default redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with admin privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + internal cas-admin redmine user => login => preserves admin account
+    Given the user is member of the admin user group
+    And the user has an internal admin redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with admin privileges
+
+  @requires_testuser
+  Scenario: cas user + internal special-admin redmine user => login => has admin access
```

Comment:
Doesnt this contradict the test "cas user + internal cas-admin redmine user => login => demote to default redmine account"?

Reply:
No, because in this case the internal admin account in Redmine was promoted by another admin account and not through the CAS login. We call this the special admin account in Redmine.

Reply:
I renamed the test to ` A regular CAS user, with existing special admin Redmine account granted internally logs into Redmine and preserves admin privileges`